src/main/kotlin/net/casual/arcade/border/CenterBorderState.kt

```diff
@@ -0,0 +1,17 @@
+package net.casual.arcade.border
+
+import net.minecraft.world.level.border.BorderStatus
+import org.joml.Vector2d
+
+interface CenterBorderState {
+
+
+
+    fun getCenter(): Vector2d
+
+    fun update(): CenterBorderState
+
+    fun onCenterChange()
+
+    fun setCenter(vector2d: Vector2d)
```

Comment:
This should be set at construction, the state should not be mutable. This method should be removed